test/test_jit.py

```diff
@@ -6577,6 +6577,60 @@ def foo(x):
 
         self.assertEqual(foo(input), input)
 
+    def test_pretty_printer(self):
```

Comment:
Can we please add a new `TestCase` for this? It's unrelated to all other functionality